modules/client-common/src/constants.ts

```diff
@@ -365,6 +403,38 @@ export const LIVE_CONTRACTS: {
       tokenVotingSetupAddress: activeContractsList.sepolia.TokenVotingSetup,
       ensRegistryAddress: activeContractsList.sepolia.ENSRegistry,
     },
+    [SupportedNetwork.ARBITRUM]: {
+      daoFactoryAddress: activeContractsList.arbitrum.DAOFactory,
```

Comment:
you can not do:
`daoFactoryAddress: activeContractsList[network].DAOFactory`
instead of repeating the same structure for each network?